src/libtorch.cc

```diff
@@ -448,8 +447,6 @@ ModelState::ParseParameters()
         TRITONSERVER_ErrorDelete(err);
       }
     } else {
-      // Override, disable NvFuser till fixed
-      enable_nvfuser = false;
       enable_nvfuser_pair_ = {true, enable_nvfuser};
       LOG_MESSAGE(
           TRITONSERVER_LOG_WARN, (std::string("NvFuser is ") +
```

Comment:
Do we need to change this from LOG_WARN to LOG_INFO?

Reply:
Sure, changed.